<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Waiting</class>
 <widget class="QWidget" name="Waiting">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>206</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>431</width>
    <height>206</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>431</width>
    <height>206</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Waiting</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../picture/title.ico</normaloff>../picture/title.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="time_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>391</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>请稍候</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="GotoWeb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(1, 166, 255);</string>
   </property>
   <property name="text">
    <string>等待时间太长，可以点击这里前往官网等待</string>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>431</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>time_label</zorder>
  <zorder>GotoWeb</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
